recipes/stuf/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set name = "stuf" %}
+{% set version = "0.9.16" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 080a88b940ca844474c239cc7aab0c530187e637a9dc6df111a99e2955bb35db
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - parse >=1.6.6
+
+test:
+  imports:
+    - stuf
+    - tests
+
+about:
+  home: https://bitbucket.org/lcrees/stuf
+  license: BSD
```

This should probably be one of these:
```suggestion
  license: BSD-2-Clause
```
```suggestion
  license: BSD-3-Clause
```